config/routes.rb

```diff
@@ -49,8 +44,13 @@
   # Video pages
   get ":slug", to: "video#show", constraints: FormatRoutingConstraint.new('video')
 
+  # Legacy done pages
+  constraints(slug: /(transaction-finished|driving-transaction-finished)/) do
+    get "/:slug(.:format)" => "completed_transaction#show"
+  end
+
   # Done pages
-  get "*slug", slug: %r{done/.+}, to: "root#publication"
+  get "*slug", slug: %r{done/.+}, to: "completed_transaction#show", constraints: FormatRoutingConstraint.new('completed_transaction')
```

Comment:
Spoke to @whoojemaflip about removing the '/done' slug constraint so that we can remove the Legacy done pages constraints from the routes.rb file. Will attempt this after https://github.com/alphagov/frontend/pull/1060 is merged